specs/_features/eip7594/das-core.md

```diff
@@ -54,6 +55,7 @@ The following values are (non-configurable) constants used throughout the specif
 | - | - | - |
 | `RowIndex` | `uint64` | Row identifier in the matrix of cells |
 | `ColumnIndex` | `uint64` | Column identifier in the matrix of cells |
+| `CustodyIndex` | `uint64` | Custody group identifier in the matrix of cells |
```

Comment:
This is a bit wrong in my opinion. It's an index into a set of groups & doesn't involve the matrix.
```suggestion
| `CustodyIndex` | `uint64` | Custody group identifier in the set of custody groups |
```